src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -110,6 +110,12 @@
     @Option(doc="Sample Size used for Theoretical Het Sensitivity sampling. Default is 10000.", optional = true)
     public int SAMPLE_SIZE=10000;
 
+    @Option(shortName = "LARGE_INTERVALS", doc = "An interval list file that contains contig-sized locations of the positions to assess. It is important that " +
+            "the intervals be chosen so that they won't have overlapping reads that extend beyond the boundaries of the interval. For smaller sites use " +
+            "CollectWgsMetricsFromSampledSites.",
+            optional = true)
```

Comment:
I think if we want this argument to be INTERVALS, then the argument in CollectWgsMetricsFromSampledSites will need to be something other than INTERVALS. I thought it was better to not change the FromSampledSites version in case there were scripts out there already using it.